### PR TITLE
Verify cloudformation

### PR DESCRIPTION
**Pull request by TheoLeanse**
/cc @matthew-andrews

### CODE REVIEW

**Comment by TheoLeanse** [comment]
`pip install`ing the python awscli package onto the ci box seems to work... but seems a bit nasty

**Comment by TheoLeanse** [comment]
screen shot 2016-09-02 at 16 26 30

File: Makefile [hunk 1]
```diff
@@ -6,3 +6,9 @@ deploy-test:
 
 deploy-prod:
 	@$(MAKE) deploy-super
+
+validate-template:
+	@pip install awscli -q
```
Comment (matthew-andrews) [inline Makefile]: can put in the `make install` stage?

**Comment by matthew-andrews** [comment]
a thing of beauty.

cc @phamann @richard-still-ft will be very excited

File: Makefile [hunk 2]
```diff
@@ -1,14 +1,18 @@
 include n.Makefile
 
-deploy-test:
-	@export AWS_ACCESS_KEY=$(AWS_ACCESS_KEY_TEST); export AWS_SECRET_KEY=$(AWS_SECRET_KEY_TEST); $(call CONFIG_VARS,production) > $(APEX_PROD_ENV_FILE) && apex deploy --env-file $(APEX_PROD_ENV_FILE) --env test
-	@rm $(APEX_PROD_ENV_FILE) && $(DONE)
+install:
+	@$(MAKE) install-super
+	@pip install awscli -q
```
Comment (matthew-andrews) [inline Makefile]: 👌

**Pull request merged**